go/flags/endtoend/vttablet.txt

```diff
@@ -300,6 +300,7 @@ Flags:
       --restore-to-timestamp string                                      (init incremental restore parameter) if set, run a point in time recovery that restores up to the given timestamp, if possible. Given timestamp in RFC3339 format. Example: '2006-01-02T15:04:05Z07:00'
       --restore_concurrency int                                          (init restore parameter) how many concurrent files to restore at once (default 4)
       --restore_from_backup                                              (init restore parameter) will check BackupStorage for a recent backup at startup and start there
+      --restore_from_backup_allowed_engines strings                      if present will filter out any backups taken with engines not included in the list
```

Comment:
New flags should use dashes, not underscores. Sorry, we haven't got around to deprecating and replacing the old-style flags, so we have this pitfall where you follow what looks like the standard pattern, but it isn't.

Reply:
For future reference, the vitess-bot checklist is helpful here. It tells you what is expected under `New flags`.

Reply:
aha, thanks @deepthi! I will update it